src/persist/src/indexed/mod.rs

```diff
@@ -158,20 +158,18 @@ impl Pending {
 /// Diff)` updates.
 ///
 /// The lifecycle of contained entries is as follows:
-/// - Initially: inserted into an [Unsealed], which indexes them by
-///   `(time, key, value)`.
-/// - Once the update's time has been "seal"ed: transferred from the
-///   [Unsealed] into a [Trace], which indexes them by `(key, value,
-///   time)`.
+/// - Initially: inserted into an [Unsealed], which indexes them by `(time, key,
```

Comment:
oops I forgot to update this comment that Unsealed no longer indexes anything

Reply:
I'll toss in a commit